### ISSUE TITLE
ropper cannot find exec chain in calc.exe

### ISSUE THREAD

**Issue by techvoltage** [sashs/Ropper#62]
ropper --file calc.exe --chain execve
[INFO] Load gadgets from cache
[LOAD] removing double gadgets... 100%

[INFO] ROPchain Generator for syscall execve:

[INFO]
write command into data section
eax 0xb
ebx address to cmd
ecx address to null
edx address to null

[ERROR] 'PE' object has no attribute 'sections'
[ERROR] Please report this error on https://github.com/sashs/ropper

**Comment by sashs** [comment]
Hi, 

thanks for reporting. I will add a check if the current file is an ELF file
It makes no sense to create an execve chain which is for linux with gadgets found in a PE file. :)

**Comment by eternaleclipse** [comment]
I've fixed this on my machine, waiting for #83 to be fixed and then PR.

The fix is quite simple, just add:
```python
if isinstance(self._binaries[0], ropper.loaders.pe.PE):
    raise RopChainError("Cannot create '{}' chain for PE files".format(self.name()))
```

To the beginning of the following functions:
```
ropper/ropchain/arch/ropchainx86.py:RopChainX86System.create
ropper/ropchain/arch/ropchainx86.py:RopChainX86Mprotect.create

ropper/ropchain/arch/ropchainx86_64.py:RopChainMprotectX86_64.create
ropper/ropchain/arch/ropchainx86_64.py:RopChainSystemX86_64.create
```

**Comment by sashs** [comment]
I've fixed this is a different way. https://github.com/sashs/Ropper/commit/06a2c1b3e16099078237f57e51ace46f4f67fe60
